content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -75,59 +145,169 @@ Based on resources the task has used, Datadog assigns the appropriate portion of
+[105]: https://cloud.google.com/kubernetes-engine/docs/how-to/cost-allocations#enable_breakdown
+
+{{% /tab %}}
+{{< /tabs >}}
 
 ## Understanding spend
 
-Using the `allocated_spend_type` tag, you can visualize the spend category associated with your cost at the Kubernetes node, AWS ECS host, AWS EBS volume, or cluster level. Costs are allocated into different spend types:
+Use the `allocated_spend_type` tag to visualize the spend category associated with your costs at various levels, including the Kubernetes node, container orchestration host, storage volume, or entire cluster level.
+
+{{< tabs >}}
+{{% tab "Azure" %}}
+
+### Compute
+
+The cost of a host instance is split into two components: 60% for the CPU and 40% for the memory. Each component is allocated to individual workloads based on their resource reservations and usage.
+
+Costs are allocated into the following spend types:
 
 | Spend type | Description    |
 | -----------| -----------    |
-| Managed service fee | Cost of associated fees charged by the provider for managing the cluster, such as EKS or GKE fees. |
-| Usage | Cost of resources used by workloads. |
-| Workload idle | Cost of resources that are reserved and allocated but not used by workloads. |
-| Cluster idle | Cost of resources that are not reserved by workloads in a cluster. |
+| Managed service fee | Cost of associated fees charged by the provider for managing the cluster, such as Azure fees. |
+| Usage | Cost of resources (such as memory and CPU) used by workloads, based on the average usage on that day. |
+| Workload idle | Cost of resources (such as memory and CPU) that are reserved and allocated but not used by workloads. This is the difference between the total resources requested and the average usage. |
+| Cluster idle | Cost of resources (such as memory and CPU) that are not reserved by workloads in a cluster. This is the difference between the total cost of the resources and what is allocated to workloads. |
+
+{{% /tab %}}
+{{% tab "AWS" %}}
 
 ### Compute
 
 The cost of a host instance is split into two components: 60% for the CPU and 40% for the memory. Each component is allocated to individual workloads based on their resource reservations and usage.
 
-- Usage: Cost of memory and CPU used by workloads, based on the average usage on that day.
-- Workload idle: Cost of memory and CPU that is being reserved and allocated but not used. This is the difference between the total resources requested and the average usage.
-- Cluster idle: Cost of memory and CPU not reserved by workloads in a cluster. This is the difference between the total cost of the resources and what is allocated to workloads.
+Costs are allocated into the following spend types:
 
-### Persistent volumes (AWS only)
+| Spend type | Description    |
+| -----------| -----------    |
+| Managed service fee | Cost of associated fees charged by the provider for managing the cluster, such as AWS ECS fees. |
```

Comment:
Managed service fee should probably be in their own section unrelated to compute costs (this goes for all 3 clouds)
I think we should move managed service fees up to a type of resource personally. I would put it as the last resource in the list for each cloud.

Also small typo for each cloud the such as should be: AWS -> EKS, Azure -> AKS, GCP -> GKE